src/lib/input/input-container.scss

```diff
@@ -3,70 +3,124 @@
 @import '../core/style/form-common';
 
 
-$mat-input-floating-placeholder-scale-factor: 0.75 !default;
-$mat-input-wrapper-spacing: 1em !default;
-$mat-input-hint-min-space: 10px !default;
+// Min amount of space between start and end hint.
+$mat-input-hint-min-space: 1em !default;
+// The amount of additional line space beyond 1em on the input (1/2 below, 1/2 above).
+$mat-input-line-spacing: 0.125 !default;
+// The height of the underline.
+$mat-input-underline-height: 1px !default;
+// The amount to scale the font for the floating label and subscript.
+$mat-input-helper-font-scale: 0.75 !default;
```

Comment:
it's the terminology used in the spec, changed it to subscript to match our terminology